src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
I think the localization is working (it says "hallo" instead of version because I changed the string in my en_US locale and reloaded the extension to show the new localization).

We should update the locale string however to support languages with different word orders. This means using placeholders (https://github.com/EFForg/privacybadger/issues/1329), which I'm not sure our current localization setup supports. This isn't a blocker though.